### DISCUSSION TITLE
Controlled Mode for Panel Component

### DISCUSSION THREAD

**Discussion by trombonist76** [primefaces/community#489] [PrimeVue] [answered]
Hi PrimeVue Team,

I'd like to request a controlled mode for the Panel component. This feature would allow external management of the Panel's open/close state, offering enhanced flexibility in various use cases. It aligns with common practices in UI component libraries and would be a valuable addition.

Thanks,

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

There is `collapsed` property whose default value is false. https://primevue.org/panel/#api.panel.props.collapsed
